### PR TITLE
Adds effects

### PR DESCRIPTION
**Pull request by Rocker2102**

### CODE REVIEW

File: assets/css/style.css [hunk 1]
```diff
@@ -178,7 +178,19 @@ h2 {
     color: #ccc;
 }
 
+.txt-type {
+    color: rgba(255, 255, 0, 0.8);
+}
```
Comment (prateekKrOraon) [inline assets/css/style.css]:
```suggestion
.txt-type {
    color: rgba(255, 255, 0, 0.8);
}
```

try using these colors #FFA726, #FFC107 and #FF5722 
same color for both text and cursor. Choose any of the three. Whichever looks good.

File: assets/css/style.css [hunk 2]
```diff
@@ -178,7 +178,19 @@ h2 {
...
+
 /* Cursor */
-.txt-type > .txt {
-    border-right: 0.1rem solid #ccc;
+.txt-type::after {
+    color: #fff;
+    content: "_";
+    animation: animate-cursor 1s infinite;
+}
```
Comment (prateekKrOraon) [inline assets/css/style.css]: Here also the same color as that of the text.

File: assets/css/style.css (hunk 1 above)
Comment (prateekKrOraon) [reply assets/css/style.css]: @Rocker2102

File: assets/css/style.css (hunk 2 above)
Comment (prateekKrOraon) [reply assets/css/style.css]: @Rocker2102

**Review by prateekKrOraon — CHANGES_REQUESTED** [review]
Change the colors as mentioned above.

File: assets/css/style.css (hunk 2 above)
Comment (Rocker2102) [reply assets/css/style.css:190]: **`#FF5722`** looks the best.

- [ ] color-1
- [ ] color-2
- [x] color-3

**Comment by Rocker2102** [comment]
**`#FF5722`** looks the best.

- [ ] color-1
- [x] color-3
- [ ] color-2

**Pull request merged**